geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSet.java

```diff
@@ -818,15 +781,11 @@ protected int sizeValue(OrderedSetEntry value) {
 
     public void toData(DataOutput out) throws IOException {
       InternalDataSerializer.writePrimitiveInt(size(), out);
-      final int maxIndex = getMaxIndex();
-      for (int pos = 0; pos < maxIndex; ++pos) {
-        OrderedSetEntry value = getValueAtIndex(pos);
-        if (value != null) {
-          byte[] member = value.getMember();
-          byte[] score = value.getScoreBytes();
-          InternalDataSerializer.writeByteArray(member, out);
-          InternalDataSerializer.writeByteArray(score, out);
-        }
+      for (Map.Entry<byte[], OrderedSetEntry> entry : entrySet()) {
+        byte[] member = entry.getKey();
+        double score = entry.getValue().getScore();
+        InternalDataSerializer.writeByteArray(member, out);
+        InternalDataSerializer.writeDouble(score, out);
```

Comment:
Prefer `DataSerializer` instead of `InternalDataSerializer`

Reply:
sorry, thought I'd changed that